src/Integration/RequestIntegration.php

```diff
@@ -96,17 +96,18 @@ public static function applyToEvent(self $self, Event $event, ?ServerRequestInte
         }
 
         if ($self->options->shouldSendDefaultPii()) {
-            if ($request->hasHeader('REMOTE_ADDR')) {
-                $requestData['env']['REMOTE_ADDR'] = $request->getHeaderLine('REMOTE_ADDR');
+            $server = $request->getServerParams();
+            if (isset($server['REMOTE_ADDR'])) {
```

Comment:
Please add a blank line above this one to improve readability

Reply:
Done